client_util/Cargo.toml

```
@@ -8,7 +8,7 @@ license.workspace = true
 
 [dependencies]
 http = "0.2.9"
-reqwest = { version = "0.11", default-features = false, features = ["stream", "rustls-tls"] }
+reqwest = { version = "0.11", default-features = false, features = ["stream", "rustls-tls", "rustls-tls-native-roots"] }
```

```suggestion
reqwest = { version = "0.11", default-features = false, features = ["stream", "rustls-tls-native-roots"] }
```

since `rustls-tls` is "use rustls for TLS but I don't care about the CA backend" and that defaults to `rustls-tls-webpki-roots`.

same goes for the other changes.